corehq/apps/cleanup/management/commands/delete_forms.py

```diff
@@ -0,0 +1,52 @@
+from django.core.management.base import BaseCommand
+import csv
+import itertools
+from dimagi.utils.chunked import chunked
+from corehq.form_processor.models import XFormInstance
+
+
+INDEX_FORM_ID = 1
+CHUNK_SIZE = 100
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument('domain', help='Domain name that owns the forms to be deleted')
+        parser.add_argument('filename', help='path to the CSV file')
+        parser.add_argument('--resume_id', help='form ID to start at, within the CSV file')
+
+    def handle(self, domain, filename, resume_id=None, **options):
+        # expects the filename to have a CSV with a header containing "Deletion Status" and "Form ID" fields
```

Comment:
Do we actually use the "Deletion Status" field? I only see the "Form ID" field being used. My thinking here is if there is a Deletion Status field with values under it, someone's going to expect those values to be used for something, and if we're _not_ going to use them, then we should require the CSV not have that column.

Reply:
This is working with the CSV that Enveritas submitted to us. If I simplify this script to expect a sheet without that column, then I'd need to write another script to port the Envertias CSV over. Do you think that's something we should do long term?

Reply:
Addressed in 4fccff9483682f5ae3f13ae902db5b8fa069ccbe.  As mentioned offline, I think the process going forward should be that customers making this request provide us a CSV file that only contains a single column representing form ID. In this case, the customer provided an initial column called "Deletion Status" which only seems to contain `ready_to_request`. However, if one of the rows contained something like `do_not_delete` instead, we wouldn't want to have the responsibility for interpreting that field.